### ISSUE TITLE
Gem version 0.40.3 goto definition issue

### ISSUE THREAD

**Issue by andersennl** [castwide/solargraph#411]
Goto definition stopped working with version 0.40.3 using VSCode. Going back to version 0.40.2 works again.

I couldn't find any error log output in VSCode. Are there any more details I can provide to help?

Thanks for your work!

**Comment by nomasprime** [comment]
Works with NeoVim and CoC.

**Comment by castwide** [comment]
I haven't encountered this problem either.

@andersennl Can you provide a reproducible example?

**Comment by andersennl** [comment]
I just reinstalled gem version 0.40.3 but was unable to reproduce it. When I created the issue, I made several attempts, but VSCode did not show any relevant functionality (restarted VSCode, restarted Solargraph language server). Uninstalling gem version 0.40.3 and using version 0.40.2 instantly made "go to definition" in VSCode work again, hence I created the issue.

So sorry for the confusion. As I said, I couldn't find any logs, so I still can't tell why VSCode stopped displaying anything. In fact, I have this problem regularly but restarting the Solargraph language server in VSCode usually fixes the problem. Obviously this did not work the last time. I'll reopen the issue in case I'm able to get any more insights.
